### PR TITLE
MLPAB-1777 Show when notices of intent to register have been sent

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

File: cmd/event-received/makeregister_event_handler.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		if err != nil {
+			return err
+		}
+
+		uidClient := factory.UidClient()
+
+		return handleUidRequested(ctx, uidStore, uidClient, cloudWatchEvent)
+
+	default:
+		return fmt.Errorf("unknown cloudwatch event")
```
Comment (acsauk) [inline cmd/event-received/makeregister_event_handler.go:28]: Should this be updated to `makeregister` event now we're splitting by service?

File: internal/page/lpa_progress_tracker.go [hunk 2]
```diff
@@ -45,16 +64,16 @@ func (pt ProgressTracker) Progress(lpa *lpastore.Lpa) Progress {
...
 			"lpaRegistered":             pt.Localizer.T("theLpaHasBeenRegistered"),
 		}
 	} else {
 		labels = map[string]string{
-			"paid":                   "",
-			"confirmedID":            "",
 			"donorSigned":            pt.Localizer.T("youveSignedYourLpa"),
 			"attorneysSigned":        pt.Localizer.Count("attorneysHaveDeclared", len(lpa.Attorneys.Attorneys)),
 			"lpaSubmitted":           pt.Localizer.T("weHaveReceivedYourLpa"),
+			"noticesOfIntentSent":    "weSentAnEmailYourLpaIsReadyToRegister",
```
Comment (acsauk) [inline internal/page/lpa_progress_tracker.go:76]: Did we come to a decision yet on status changes vs notification sent for showing this?

**Review by acsauk — COMMENTED** [review]
A couple of questions 💬

File: internal/page/lpa_progress_tracker.go (hunk 2 above)
Comment (hawx) [reply internal/page/lpa_progress_tracker.go:76]: Sirius will change the status in lpa-store, that will then cause an event for us? (Is that what you mean?)

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**